Client/src/Pages/Status/CreateStatus/index.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from "prop-types";
+import { useNavigate } from "react-router";
+import { useEffect, useState } from "react";
+import { Box, Tab, useTheme } from "@mui/material";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import GeneralSettingsPanel from "../../../Components/TabPanels/Status/GeneralSettingsPanel";
+import {
+	capitalizeFirstLetter,
+	toLowerCaseFirstLetter,
+} from "../../../Utils/stringUtils";
+import ContentPanel from "../../../Components/TabPanels/Status/ContentPanel";
+
+/**
+ * CreateStatus page renders a page with tabs for general settings and contents.
+ * @param {string} [props.open] - Specifies the initially open tab: 'general settings' or 'content'.
+ * @returns {JSX.Element}
+ */
+
+const CreateStatus = ({ open = "general-settings" }) => {
+	const theme = useTheme();
+	const navigate = useNavigate();
+	let tabList = ["General Settings", "Contents"];
+	const tab = open
+		.split("-")
+		.map((a) => capitalizeFirstLetter(a))
+		.join(" ");
+
+	const [tabIdx, setTabIdx] = useState(tabList.indexOf(tab));
+
+	const handleTabChange = (event, newTab) => {
+		let toNavString = newTab.split(" ").map((a) => toLowerCaseFirstLetter(a));
+		setTabIdx(tabList.indexOf(newTab));
+		toNavString = toNavString.join("-");
+		navigate(`/status/${toNavString}`);
+	};
```

Comment:
Here, instead of having event and newTab, we can just have event, and pass a name to the inputs. We then destructure the name from event.target.name.
Let's try to keep the name the same as the param, to avoid the text transformation back and forth

Reply:
event and newTab are all passed down from onChange of Tablist

Reply:
I see that, I am suggesting we do not have the new tab anymore, and we get that information from event.target.name

Reply:
Here we are discussing onChange x name or value. Do you see any problem with using the name for that? Otherwise it would be best to follow that for consistency

Reply:
@jennifer-gan I think what @marcelluscaio suggests here makes sense, `newTab` is indeed passed to the handler, but in our case it makes more sense to use the event since we need a specific value that is not the same as `newTab`.

We gain two benefits

1.  Consistency with how changes are handled with other pages where we use `event.target.name`
2.  Avoiding text transformation - The reason for the transformation won't be clear to newcomers to the project and is an extra step we can avoid.

I think it's worth the change unless you have a strong objection to using the `name` property?